clang/include/clang/CIR/Dialect/IR/CIROps.td

```diff
@@ -1498,12 +1509,23 @@ def GetBitfieldOp : CIR_Op<"get_bitfield"> {
     different storages. The `cir.get_bitfield` operation gets the value
     of the bitfield
     ```C++
+    typedef struct {
+      int a : 4;
+      int b : 27;
+      int c : 17;
+      int d : 2;
+      int e : 15;
+    } S;
+
     int load_bitfield(S& s) {
       return s.d;
     }
     ```
 
     ```mlir
+    // 'd' is in the storage with the index 1
+    !ty_22S22 = !cir.struct<struct "S" {!cir.int<u, 32>, !cir.int<u, 32>, !cir.int<u, 16>, !cir.int<u, 32>} #cir.record.decl.ast>
```

Comment:
This is great, thanks for adding more info on the type. One minor nit: for clarity, rewrite to something like this:
`!struct_with_bitfields = !cir.struct<struct "S" {!cir.int<u, 32>, !cir.int<u, 32>, ...>`

Reply:
well, maybe let's write just `struct_type`. The are several words like `bitfield` around, so the context is already set. What do you think?